manual/sourcetree-disableautoupdate/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,4 @@
+﻿$ErrorActionPreference = 'Stop'
+
+# Enable auto update
+Remove-ItemProperty -Name 'AutoUpdatePolicy' -Path 'HKCU:\Software\Caphyon\Advanced Updater\{138B87CA-4905-43FE-BB63-184FFB0CE1EC}\Settings'
```

Comment:
as per [2.2 Formatting](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#22-formatting), you're missing an empty line at the end of the file.